src/components/chart/state-history-chart-line.ts

```diff
@@ -54,8 +54,6 @@ export class StateHistoryChartLine extends LitElement {
 
   @property({ attribute: "fit-y-data", type: Boolean }) public fitYData = false;
 
-  @property({ attribute: false, type: Number }) public height?: number;
```

Comment:
Should we remove the height option completely? I think we can keep this?

Reply:
It wasn't there before so isn't used. I didn't want to add extra attributes without a use case. We can always add it if we need it.